src/OpenLoco/src/OpenLoco.cpp

```diff
@@ -748,6 +748,7 @@ namespace OpenLoco
             auto freq = Config::get().autosaveFrequency;
             if (freq > 0 && _monthsSinceLastAutosave >= freq)
             {
+                _monthsSinceLastAutosave = 0;
                 autosave();
                 autosaveClean();
```

Comment:
Nice catch! Probably best to handle it through `autosaveReset()` though, e.g.:

```suggestion
                autosave();
                autosaveClean();
                autosaveReset();
```

Reply:
Changed _monthsSinceLastAutosave = 0; to autosaveReset(); per review suggestion